conversions/time_conversions.py

```diff
@@ -0,0 +1,57 @@
+"""
+A unit of time is any particular time interval, used as a standard
+way of measuring or expressing duration.
+The base unit of time in the International System of Units (SI),
+and by extension most of the Western world, is the second,
+defined as about 9 billion oscillations of the caesium atom.
+
+WIKI: https://en.wikipedia.org/wiki/Unit_of_time
+"""
+
+time_chart: dict[str, float] = {
+    "seconds": 1.0,
+    "minutes": 60.0,  # 1 minute = 60 sec
+    "hours": 3600.0,  # 1 hour = 60 minutes = 3600 seconds
+    "days": 86400.0,  # 1 day = 24 hours = 1440 min = 86400 sec
+    "weeks": 604800.0,  # 1 week=7d=168hr=10080min = 604800 sec
+}
+
+time_chart_inverse: dict[str, float] = {
+    "seconds": 1.0,
+    "minutes": 1 / 60.0,  # 1 minute = 1/60 hours
+    "hours": 1 / 3600.0,  # 1 hour = 1/3600 days
+    "days": 1 / 86400.0,  # 1 day = 1/86400 weeks
+    "weeks": 1 / 604800.0,  # 1 week = 1/604800 seconds
+}
```

Comment:
Let's calculate `time_chart_inverse`
```suggestion
time_chart_inverse: dict[str, float] = {key: 1 / value for key, value in time_chart.items()}
```